src/List/ListItem.js

```diff
@@ -0,0 +1,46 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import ListItemContent from './ListItemContent';
+
+class ListItem extends React.Component {
+    static propTypes = {
+        className: PropTypes.string,
+        twoLine: PropTypes.bool,
+        threeLine: PropTypes.bool
+    };
+
+    static defaultProps = {
+        twoLine: false,
+        threeLine: false
+    };
+
+    render() {
+        const { className, twoLine, threeLine, ...otherProps } = this.props;
+
+        const classes = classNames('mdl-list__item', {
+            'mdl-list__item--two-line': twoLine,
```

Comment:
`twoLine && !threeLine`.
same on the line below

(I'm still unsure about using a trailing `s` or not. Feels weird to not have it though :/)